layers/rebox/README.org

```diff
@@ -0,0 +1,368 @@
+#+TITLE: README for rebox layer
+#+AUTHOR: Christian E. Hopps
+#+EMAIL: chopps@gmail.com
+#+STARTUP: indent
+
+
+#+TITLE: Rebox layer
+#+HTML_HEAD_EXTRA: <link rel="stylesheet" type="text/css" href="../../../css/readtheorg.css" />
+
+* Table of Contents                                         :TOC_4_org:noexport:
+ - [[Installation][Installation]]
+ - [[Commands][Commands]]
+   - [[spacemacs bindings][spacemacs bindings]]
+   - [[rebox-minor-mode bindings][rebox-minor-mode bindings]]
+ - [[Configuration][Configuration]]
+   - [[Example configuration][Example configuration]]
+   - [[Style Templates][Style Templates]]
+     - [[Standard packaged templates][Standard packaged templates]]
+       - [[Leader Comment Character Templates (lisp, python, perl, ...)][Leader Comment Character Templates (lisp, python, perl, ...)]]
+       - [[Text Mode (config required)][Text Mode (config required)]]
+       - [[C Mode (disabled due to bug)][C Mode (disabled due to bug)]]
+     - [[Templates added by layer][Templates added by layer]]
+
+* Installation
+* Commands
+** spacemacs bindings
+
+| Keybinding | Command                 |
+|------------+-------------------------|
+| SPC c b    | rebox-dwim              |
+| SPC c B    | rebox-dwim-no-fill      |
+|------------+-------------------------|
+
+** rebox-minor-mode bindings
+
+|----------+-------------------------|
+| S-return | rebox-indent-new-line   |
+| M-q      | rebox-dwim              |
+| M-Q      | rebox-cycle             |
+| C-a      | rebox-beginning-of-line |
+| C-e      | rebox-end-of-line       |
+| C-k      | rebox-kill-line         |
+| M-w      | rebox-kill-ring-save    |
+| C-y      | rebox-yank              |
+| M-y      | rebox-yank-pop          |
+| M-c      | rebox-center            |
+| SPC      | rebox-space             |
+| DEL      | rebox-backspace         |
+
```

Comment:
Okay. Understood. Then this might need some evilfication as well. As a suggestion, it can go under `SPC c R` with evilfied mnemonics. `$` end of line, `0` beginning of line et al.?